src/steth/usdc-adapters/IAdapter.sol

```diff
@@ -0,0 +1,76 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+pragma solidity ^0.8.19;
+
+import {Constants as C} from "../../lib/Constants.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {WETH} from "solmate/tokens/WETH.sol";
+
+/**
+ * @notice Interface for adapters that allow interactions with the lending protocols
+ */
+abstract contract IAdapter {
+    ERC20 constant usdc = ERC20(C.USDC);
+    WETH constant weth = WETH(payable(C.WETH));
+
+    /**
+     * @notice Returns the adapter's ID
+     */
+    function id() external virtual returns (uint8);
```

Comment:
You should make this uint256 since the EnumerableMap in scWETHv2 is using uint256. I don't see any value in keeping it uint8.

Reply:
[changed](https://github.com/lindy-labs/sandclock-contracts/pull/58/commits/a4e230ff913c82d72a59f15cebc4ab7503518201)